### PR TITLE
[BACKEND] Support events and attendance records

### PR DESCRIPTION
**Pull request by elvincheng3**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+version: "3.5"
```
Comment (justagoodfriend) [inline docker-compose.yml:1]: what are we containerizing with this docker-compose?

File: api/member/getMember.ts [hunk 2]
```diff
@@ -8,7 +8,8 @@ const membersController = new MembersController();
 const getMember = async function (req: VercelRequest, res: VercelResponse) {
   try {
     const member = await membersController.getMember(req.query.id as string);
-    res.status(200).json({ member: member });
+    console.log(member);
```
Comment (justagoodfriend) [inline api/member/getMember.ts:11]: nit could we clean this up?

File: api/event/getAllEvents.ts [hunk 3]
```diff
@@ -14,4 +14,4 @@ const getAllEvents = async (req: VercelRequest, res: VercelResponse) => {
   }
 };
 
-export default allowCors(getAllEvents);
+export default allowCors(getAllEvents, true);
```
Comment (justagoodfriend) [inline api/event/getAllEvents.ts:17]: nit: is this necessary if the cors middleware defaults to true

File: src/controllers/recordController.ts [hunk 4]
```diff
@@ -10,23 +10,36 @@ import {
 export class RecordController {
   async getRecordForMember(id: string) {
     const [data] = await pool.query(
-      `SELECT * FROM AttendanceRecord WHERE person_id = ?`,
+      `SELECT * FROM AttendanceRecord JOIN Event ON AttendanceRecord.event_id = Event.uuid WHERE person_id = ?`,
```
Comment (justagoodfriend) [inline src/controllers/recordController.ts:13]: Good idea combining the event object onto the record to save the api call to fetch that data

File: docker-compose.yml (hunk 1 above)
Comment (elvincheng3) [reply docker-compose.yml:1]: Deleting

**Review by justagoodfriend — APPROVED** [review]

**Pull request merged**